lib/created_message_queue_handler.ts

```diff
@@ -17,6 +17,8 @@ import { configureAzureContextTransport } from "./utils/logging";
 
 import * as documentDbUtils from "./utils/documentdb";
 
+import { Set } from "json-set-map";
```

Comment:
We have [this io-ts set](https://teamdigitale.github.io/italia-ts-commons/modules/_types_.html#readonlysettype)

Reply:
yes, but that's  not what we use in other models (ie. service with authorized_cidrs). we may consider a broader refactor using io-ts sets instead of this one (but we must check that io-ts sets are serializable first)